frontend/src/components/manageUsers/ManageUsersPageContent.tsx

```diff
@@ -0,0 +1,49 @@
+import { getSession } from "src/services/auth/session";
+import { getUserOrganizations } from "src/services/fetch/fetchers/organizationsFetcher";
+import { Organization } from "src/types/applicationResponseTypes";
+
+import { getTranslations } from "next-intl/server";
+import { ErrorMessage, GridContainer } from "@trussworks/react-uswds";
+
+import { ManageUsersBreadcrumbs } from "src/components/manageUsers/ManageUsersBreadcrumbs";
+import { PageHeader } from "src/components/manageUsers/PageHeader";
+
+export async function ManageUsersPageContent({
+  organizationId,
+}: {
+  organizationId: string;
+}) {
+  const t = await getTranslations("ManageUsers");
+
+  const session = await getSession();
+  if (!session?.token) {
+    return (
+      <GridContainer className="padding-top-2 tablet:padding-y-6">
+        <ErrorMessage>{t("errors.notLoggedInMessage")}</ErrorMessage>
+      </GridContainer>
+    );
+  }
+  let userOrganizations: Organization[] | undefined;
+  try {
+    const [organizationsResult] = await Promise.all([
+      getUserOrganizations(session.token, session.user_id),
```

Comment:
it probably makes more sense to get the data from the organization details endpoint rather than the userOrgs endpiont - that way we don't have to filter, and it's possible the org details will give us more info we'll need later on